test/e2e/specs/editor/plugins/iframed-enqueue-block-editor-settings.spec.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'iframed block editor settings styles', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activatePlugin(
```

Comment:
The original test enables `emptytheme` here. Do you need to do it here too?

Reply:
Not anymore. Since then, all editors have been iframed by default. Unless there's a `v2` block or metabox fields enabled.